README.md

```diff
@@ -14,23 +14,73 @@
   </a>
 </a>
 </p>
+
+<h2 align="center"><a href="https://killercoda.com/open-feature/scenario/flagd-demo" target="_blank">Try flagd in the Killercoda Playground (in browser)</a></h4>
 <!-- markdownlint-enable MD033 -->
 
 ## Features
 
+Flagd is a simple command line tool for fetching and evaluating feature flags for services.
+
+Think of it as a ready-made, open source, OpenFeature compliant feature flag backend system.
+
 - OpenFeature compliant with providers available in many languages
-- Multiple flag configuration sources including `files`, `http`, and `Kubernetes`
+- Multiple flag configuration sources including `files`, `http`, and `Kubernetes` can be used simultaneously
 - Accessible over gRPC and HTTP
 - Supports subscriptions to real-time flag change events
 - Flexible targeting rules based on [JSON Logic](https://jsonlogic.com/)
 - Lightweight daemon, with an emphasis on performance
 - Native support for metrics using Prometheus
 
-## Get started
+## Quick Start
 
-Flagd is a simple command line tool for fetching and evaluating feature flags for services.
-It is designed to conform with the OpenFeature specification.
-To get started, follow the installation instructions in the [docs](https://github.com/open-feature/flagd/blob/main/docs/README.md).
+![logical architecture of flagd](docs/images/flagd-logical-architecture.jpg)
+*High level, logical architecture of flagd*
+
+1. flagd can be run as a standalone-binary or container. [Download and install flagd or run it as a container](docs/usage/installation_options.md)
+
+2. Start flagd:
+
+    ```sh
+    // Start flagd
+    flagd start \
+      --port 8013 \
+      --uri https://raw.githubusercontent.com/open-feature/flagd/main/samples/example_flags.flagd.json
+
+    /* Or with docker:
+      docker run \
+        --rm -it \
+        --name flagd \
+        -p 8013:8013 \
+        ghcr.io/open-feature/flagd:latest start \
+        --uri https://raw.githubusercontent.com/open-feature/flagd/main/samples/example_flags.flagd.json
+    */
+    ```
+
+    `--uri` can be a local file or any remote endpoint. Use `file:` prefix for local files. eg. `--uri file:/path/to/example_flags.flagd.json`. `gRPC` and `http` have their own requirements. More information can be found [here](docs/configuration/configuration.md#uri-patterns).
+
+    Multiple `--uri` parameters can be specified. In other words, flagd can retrieve flags from multiple sources simultaneously.
+
+3. Flagd is now ready to perform flag evaluations over either `HTTP(s)` or `gRPC`. This example utilizes `HTTP` via `cURL`.
+
+    ```sh
+    // Retrieve a String flag
+    curl -X POST "http://localhost:8013/schema.v1.Service/ResolveString" \
+      -d '{"flagKey":"myStringFlag","context":{}}' -H "Content-Type: application/json"
+
+    // Result:
+    {"value":"val1","reason":"DEFAULT","variant":"key1"}
+    ```
+
+    Updates to the underlying flag store (e.g. JSON file) are reflected by flagd in realtime. No restarts required.
```

Comment:
I wonder if step one should be to download the config file locally. That would allow the user to modify the flag config.